src/fmgc/src/efis/NearbyFacilities.ts

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2021 FlyByWire Simulations
+// SPDX-License-Identifier: GPL-3.0
+
+import { NearestSearchType } from "@fmgc/types/fstypes/FSEnums";
+import { NearestSearch, RawAirport, RawIntersection, RawNdb, RawVor } from "@fmgc/types/fstypes/FSTypes";
+
+// WARNING: this is a temporary implementation until the new nav database is complete
+// Do not write any code which depends on it
+export class NearbyFacilities {
+    private listener;
+    private initDone = false;
+
+    public nearbyAirports: Map<string, RawAirport> = new Map();
+    public nearbyNdbNavaids: Map<string, RawNdb> = new Map();
+    public nearbyVhfNavaids: Map<string, RawVor> = new Map();
+    public nearbyWaypoints: Map<string, RawIntersection> = new Map();
+    public version: number = 0;
+
+    private airportSessionId: number;
+    private ndbSessionId: number;
+    private vorSessionId: number;
+    private waypointSessionId: number;
+
+    private lastPpos = { lat: 0, long: 0 };
+    private throttler = new UpdateThrottler(10000);
+    private radius = 381 * 1852; // metres
+    private limit = 160;
```

Comment:
**suggestion:** Move those factors to `MathUtils`

Reply:
We should do that in the new nav database, which will entirely replace this file.

Reply:
Hm? Unit conversions are not really related to the nav database.

Reply:
I'm talking about whatever part of `381 * 1852` is a unit factor

Reply:
I'd prefer to not do this in this PR, and instead bring in `uom-ts` and a suitable set of units defined on top of that in a future PR.